src/locales/fr.json

```diff
@@ -31,16 +31,16 @@
       "after_created": "Une fois votre note créée, il vous est possible de la retrouver dans "
     },
     "SavingIndicator": {
-      "saved_just_now": "Enregistré à l'instant.",
-      "saved_secs_ago": "Enregistré il y a %{time} sec.",
-      "saved_min_ago": "Enregistré il y a %{time} min.",
-      "saved_mins_ago": "Enregistré il y a %{time} min.",
-      "saved_hour_ago": "Enregistré il y a %{time} heure.",
-      "saved_hours_ago": "Enregistré il y a %{time} heures.",
-      "saved_day_ago": "Enregistré il y a %{time} jour.",
-      "saved_days_ago": "Enregistré il y a %{time} jours.",
+      "saved_just_now": "Dernière mise à jour à l'instant.",
+      "saved_secs_ago": "Dernière mise à jour il y a %{time} sec",
+      "saved_min_ago": "Dernière mise à jour il y a %{time} min",
+      "saved_mins_ago": "Dernière mise à jour il y a %{time} min",
+      "saved_hour_ago": "Dernière mise à jour il y a %{time} heure",
+      "saved_hours_ago": "Dernière mise à jour il y a %{time} heures",
+      "saved_day_ago": "Dernière mise à jour il y a %{time} jour",
+      "saved_days_ago": "Dernière mise à jour il y a %{time} jours",
```

Comment:
Does it fit well on a very small screen?

Reply:
We don't display this on small screens yet (we will, but the question will be raised at this time when I will ask for the correct UI for small screens))